spec/spec.md

```diff
@@ -1752,20 +1827,6 @@ eyJhbGciOiJFZERTQSIsImtpZCI6ImRpZDprZXk6ejZNa2diZldhYloxeHFkVlNCQ1hzYU1UYkw4YXVw
 
 ### Developer Resources
 
-#### JSONPath implementations
-
-- **Node.js**
-    - https://www.npmjs.com/package/jsonpath
-    - https://www.npmjs.com/package/jsonpath-plus (Ships its own TS types and have newer updates)
-- **Java**
-    - https://github.com/json-path/JsonPath
-- **Kotlin**
-    - https://github.com/codeniko/JsonPathKt
-- **Python**
-    - https://github.com/kennknowles/python-jsonpath-rw
-- **Go**
-    - https://github.com/PaesslerAG/jsonpath
-
 #### JSON Schema implementations
```

Comment:
If we're excluding JsonPath implementations, shouldn't we exclude JSON Schema impls as well? If not, should we instead link to the official implementation list? https://json-schema.org/implementations.html